redis/commands/search/aggregation.py

```diff
@@ -300,6 +301,17 @@ def add_scores(self) -> "AggregateRequest":
         self._add_scores = True
         return self
 
+    def scorer(self, scorer: str) -> "AggregateRequest":
+        """
+        Use a different scoring function to evaluate document relevance.
+        Default is `TFIDF`.
+
+        :param scorer: The scoring function to use
+                       (e.g. `TFIDF.DOCNORM` or `BM25`)
```

Comment:
It makes sense to create an enum to handle different functions, in this way user will be aware of all possible values

Reply:
@vladvildanov added

Reply:
The signature should be changed as well, or it makes no sense to the enduser.

`def scorer(self, scorer: Scorers) -> "AggregateRequest"`

Reply:
done